test/util/display_SYSINFO_settings.py

```diff
@@ -1,31 +1,28 @@
-# Copyright 2016, EMC, Inc.
+'''
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
@anhou I don't like the file names either, but that's the way it has been for a long time.